xla/service/gpu/runtime/dynamic_slice_thunk.h

```diff
@@ -45,10 +45,17 @@ class DynamicSliceThunk : public Thunk {
  public:
   struct LoopIter {};
 
+  struct OffsetArray {
```

Comment:
Can you add a documentation for this struct, and the one above, LoopIter is mostly obvious, but this one not so much

Reply:
Done